fuzzing/coverage/coverage_tracer.go

```diff
@@ -146,28 +148,16 @@ func (t *CoverageTracer) CaptureState(pc uint64, op vm.OpCode, gas, cost uint64,
 
 	// If there is code we're executing, collect coverage.
 	if len(scope.Contract.Code) > 0 {
-		// We record coverage maps under a code hash to merge coverage across different deployments of a contract.
```

Comment:
So the above line checks if the thing we are calling has code. Wouldn't the precompiles bypass this check as well? Does it matter?

Reply:
I'm not even sure `CaptureState` is called on pre-compiles, but if it's empty in any case, this is fine, we can't collect anything from a pre-compile.